### DISCUSSION TITLE
Third-party logins (SSO)

### DISCUSSION THREAD

**Discussion by trostum** [sanity-io/sanity#3515] [Sanity Studio Support]
We use SSO in some of our sanity studios today, integrating with Azure/AD.
The [documentation](https://www.sanity.io/docs/third-party-login) is easy enough for v2, but how do I configure v3 for SSO?

Specifically, i’m wondering how to configure the [default-login.js](https://www.sanity.io/docs/third-party-login#78c9b12d4a21) options as there isn’t a “config” directory in the v3.

**Comment by kmelve** [comment]
Hi! I need to check in with the team on this one! Stay tuned…

**Comment by safejace** [comment]
@trostum https://beta.sanity.io/docs/platform/studio/v2-to-v3/custom-auth-providers#removing-a-default-auth-provider should work


edit: should* being the keyword lol

getting this error: 

```
Uncaught error: The requested module '/node_modules/.vite/deps/sanity.js?v=e1c45350' does not provide an export named 'createAuthStore'
```

**Comment by safejace** [comment]
not sure i follow how or why, but i needed to `import {createAuthStore} from 'sanity/_unstable'` as shown here: https://beta.sanity.io/docs/reference/sanity/3.0.0-dev-preview.15/_unstable/createAuthStore

**Comment by abovedave** [comment]
For anyone else lost, I found the new docs here: https://www.sanity.io/docs/auth-api-reference

**Reply by atlasiali** [reply]
Your comment helped me.

this manual is missing projectId..
https://www.sanity.io/docs/migrating-custom-auth-providers
